src/sidebar/services/groups.js

```diff
@@ -158,17 +159,23 @@ function groups(
           };
           const profileGroupsApi = api.profile.groups.read(profileParams);
           const listGroupsApi = api.groups.list(params);
-          return Promise.all([profileGroupsApi, listGroupsApi]).then(
-            ([myGroups, featuredGroups]) =>
-              combineGroups(myGroups, featuredGroups)
-          );
+          return Promise.all([
+            profileGroupsApi,
+            listGroupsApi,
+            auth.tokenGetter(),
```

Comment:
This approach looks cleaner to me that trying to thread the auth token through the API response as I did originally with `{ includeMetadata: true }`. For consistency it might be a good idea to use the same approach in both branches.